fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -931,6 +969,7 @@ fun BottomDrawer(
     expandable: Boolean = true,
     scrimVisible: Boolean = true,
     showHandle: Boolean = true,
+    skipOpenState: Boolean = false,
```

Comment:
instead we can rename it to "swipeToClose". WDYT? skipOpenState can also be confused during the opening of drawer

Reply:
Now, we have changed the functionality and it affects the opening as well.